### PR TITLE
Sasa/minor doc adjustments

### PR DESCRIPTION
**Pull request by sasa1977**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,10 +41,7 @@ def project do
 
       # ...
     ],
-    application: [
```
Comment (obrok) [inline README.md]: Why? It's one of the easier things to forget in Elixir, especially for beginners.

File: lib/gen_socket_client/test_socket.ex [hunk 2]
```diff
@@ -9,8 +9,8 @@ defmodule Phoenix.Channels.GenSocketClient.TestSocket do
   bugs. You're instead advised to implement your own callback for the
   `Phoenix.Channels.GenSocketClient` behaviour.
 
-  Notice that the module is defined in the lib (and not in the test), which
-  allows us to reuse it in tests of other projects (such as air).
+  The module is defined in the lib (and not in the test) folder, so it can be used
+  in client projects.
```
Comment (obrok) [inline lib/gen_socket_client/test_socket.ex]: maybe "... can be used in your project for testing your own functionality."?

File: README.md (hunk 1 above)
Comment (sasa1977) [inline README.md]: In Elixir 1.4+ you don't need to specify it, due to application inference. In fact specifying the app will likely break other deps (unless you list them all in `applications`).

Comment (obrok) [inline README.md]: Didn't know this change, have to read up on this.

**Pull request merged**